src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetFormatTests.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System;
+using System.Linq;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+public class DotNetFormatTests
+{
+    const string VerifyNoChangesTestDirectoryName = "VerifyNoChanges";
+    const string FormatTestDirectoryName = "FormatTest";
```

Comment:
What do you think about using `nameof(DotNetFormatTests)` here?  The benefit is that it would better associate which test created the directory.